src/openvslam_ros.cc

```diff
@@ -45,6 +46,21 @@ void system::publish_pose() {
     pose_msg.pose.pose.position.y = trans(1);
     pose_msg.pose.pose.position.z = trans(2);
     pose_pub_->publish(pose_msg);
+
+    // Create transform message between map->camera_link
+    geometry_msgs::msg::TransformStamped map_trans;
+    map_trans.header.stamp =  node_->now();
```

Comment:
I think we need to use a future timestamp like amcl.
(ref. https://github.com/ros-planning/navigation2/blob/main/nav2_amcl/src/amcl_node.cpp#L194 )
@SteveMacenski What do you think?

Reply:
Yes, almost certainly, the main localization engine should lead the clock slightly

Reply:
It is handled at https://github.com/OpenVSLAM-Community/openvslam_ros/issues/21.